source/How-To-Guides/Use-The-Dashboard.rst

```diff
@@ -1,7 +1,96 @@
-Using use the Space ROS Dashboard
-=================================
+#############################
+Using the Space ROS Dashboard
+#############################
 
-TODO:
+Static analysis is a critical step in the formal verification required for safety-critical systems.
+The Space ROS Dashboard is a tool for aggregating and tracking issues discovered through static analysis.
+These issues are represented in `SARIF format <https://sarifweb.azurewebsites.net/>`_, which is a format for representing issues found by static analysis tools.
 
-How to use the Space ROS Dashboard, including installation of the VSCode plug-in.
+The main features of the Space ROS Dashboard are as follows:
 
+- Track the dispositioning issues, including annotating them with links to bug reports in your dispositioning service of choice
+- Compare issues across multiple builds
+- View a burn-down chart of issues over time
+
+This how-to guide will walk you through the main features of the Space ROS Dashboard.
+
+Prerequisites
+=============
+
+In order to use the Space ROS Dashboard, you will have to install `Visual Studio Code <https://code.visualstudio.com/>`_ and the `Space ROS Dashboard extension <https://marketplace.visualstudio.com/items?itemName=openrobotics.spaceros-dashboard>`_.
+
+You will also need to have SARIF data to view.
```

Comment:
Might want to clarify that this happens through a `colcon build` then a `colcon test`. I've found that I need to either `--cmake-args -DCMAKE_EXPORT_COMPILE_COMMANDS` as an extra flag to build, or `set(COLCON_EXPORT_COMPILE_COMMANDS ON)` in the CMakeLists.txt for the respective package to get Cobra to run as well.

Reply:
Addressed in ef6a1f48